### ISSUE TITLE
Show Giant Swarm logo in Grafana and Managed Apps

### ISSUE THREAD

**Issue by cokiengchiara** [giantswarm/roadmap#297]
From @JosephSalisbury in [slack](https://gigantic.slack.com/archives/CPC3M70UE/p1619447726048400).

tl;dr - it would be cool if grafana (and maybe other managed apps?) had some giant swarm branding, so if people do show it off, we get some advertisement

Some options:
* Joe checked out how to do it for grafana, and they do have whitelabelling options (see https://grafana.com/docs/grafana/latest/enterprise/white-labeling/), but that’s under enterprise licenses. possibly we can cut a custom deal with them? 
* alternative is rebuilding, but i feel like that might get us into some AGPL fun
* or just a logo elsewhere, or launching it through happa. Something. I want .... some ... thing.... 

Agree w @JosephSalisbury and @othylmann it would be cool. I think like other branding things it won't directly lead to sales or anything like that, but helps create familiarity.

@giantswarm/team-halo let's discuss when we could do this and what are some ways and level of effort needed.

**Comment by cokiengchiara** [comment]
This is sth for backlog for when we have more manpower.

**Comment by cokiengchiara** [comment]
@oponder will check it out if sth can be done with not too much work.

**Comment by JosephSalisbury** [comment]
This was a cool idea, but it's kind of impractical. Closing until someone shouts
